dev/sparktestsupport/toposort.py

```diff
@@ -0,0 +1,85 @@
+#######################################################################
+# Implements a topological sort algorithm.
```

Comment:
Ping again. @srowen @pwendell, can you comment on the `NOTICE` file considerations here?